moveit_core/sensor_manager/include/moveit/sensor_manager/sensor_manager.h

```diff
@@ -40,8 +40,8 @@
 #include <vector>
 #include <string>
 #include <moveit/macros/class_forward.h>
-#include <moveit_msgs/RobotTrajectory.h>
-#include <geometry_msgs/PointStamped.h>
+#include <moveit_msgs/msg/robot_trajectory.hpp>
+#include <geometry_msgs/msg/point_stamped.hpp>
```

Comment:
Wow, being rather ignorant of the current state of ROS2, I really wonder what should be the beautiful upside of changing includes like this, apart from making huge code-bases incompatible...

Reply:
IIRC, the rational is that this way, it's possible to have services, actions and msgs that have the same name (but I'm not 100% certain).

Reply:
^ that is correct. For example in ROS1 we had the standard of adding "Action" at the end of every action message type to avoid conflicts in messages or services, but it was decided (I have no opinion) to enforce this more rigorously through folders and namespaces